php/Terminus/Models/Site.php

```diff
@@ -332,6 +333,19 @@ public function fetch(array $options = array()) {
   }
 
   /**
+   * Re-fetches site attributes from the API
+   *
+   * @return void
+   */
+  public function fetchAttributes() {
+    $response = $this->request->simpleRequest(
+      sprintf('sites/%s/settings', $this->get('id'))
+    );
+    $this->attributes = $response['data'];
+    $this->collection->sites_cache->update((array)$response['data']);
```

Comment:
I'd be happy to make that change, but I'm afraid that it's out of the scope of this PR.